### PR TITLE
Fixes application crash with no response from device

### PR DESCRIPTION
**Pull request by uspilot**
Fixes GUI part of issue  #128 https://github.com/iceman1001/ChameleonMini-rebooted/issues/128#issue-482090183
Fixes clear of output screen after device disconnection

### CODE REVIEW

File: ChameleonMiniGUI/FrmMain.cs [hunk 1]
```diff
@@ -1329,7 +1329,7 @@ private void OpenChameleonSerialPort()
                     if (!string.IsNullOrEmpty(_firmwareVersion) && _firmwareVersion.Contains("Chameleon"))
                     {
                         _cmdExtension = string.Empty;
-                        txt_output.Text = $"[+] Success, found Chameleon Mini device on '{comPortStr}' with {_deviceIdentification} installed{Environment.NewLine}";
+                        txt_output.Text += $"[+] Success, found Chameleon Mini device on '{comPortStr}' with {_deviceIdentification} installed{Environment.NewLine}";
```
Comment (iceman1001) [inline ChameleonMiniGUI/FrmMain.cs]: We used = here in order to have an empty output box when freshly connected device.   ie, don't want to see old garbage outpu.

**Comment by iceman1001** [comment]
@bogiton  you got time to check this one?

**Comment by bogiton** [comment]
These changes to the GUI seem to be fine. We could add a clear button for the output textbox to clear the previous lines if needed.

**Comment by iceman1001** [comment]
@uspilot   Thanks!

**Pull request merged**